apps/frontend/src/app/periods/[id]/data-table.tsx

```diff
@@ -249,22 +259,32 @@ export function DataTable<TData, TValue>({
               </TableCell>
             </TableRow>
             <TableRow>
-              <TableCell colSpan={columns.length} className='pt-0'>
-                <div className='flex flex-row gap-4 w-full justify-center'>
-                  <Button onClick={() => table.firstPage()} disabled={!table.getCanPreviousPage()}>
-                    {'<<'}
-                  </Button>
-                  <Button onClick={() => table.previousPage()} disabled={!table.getCanPreviousPage()}>
-                    {'<'}
-                  </Button>
-                  <div className='flex flex-col justify-center min-w-5 items-center'>{pagination.pageIndex}</div>
-                  <Button onClick={() => table.nextPage()} disabled={!table.getCanNextPage()}>
-                    {'>'}
-                  </Button>
-                  <Button onClick={() => table.lastPage()} disabled={!table.getCanNextPage()}>
-                    {'>>'}
-                  </Button>
-                </div>
+              <TableCell colSpan={columns.length}>
+                <Pagination className='mb-2'>
+                  <PaginationContent>
+                    <div className={table.getCanPreviousPage() ? 'flex' : 'pointer-events-none opacity-50 flex'}>
+                      <PaginationItem onClick={() => table.firstPage()}>
+                        <PaginationFirst />
+                      </PaginationItem>
+                      <PaginationItem onClick={() => table.previousPage()}>
+                        <PaginationPrevious />
+                      </PaginationItem>
+                    </div>
+                    <PaginationItem>
+                      <PaginationLink href='#' isActive>
+                        {pagination.pageIndex + 1}
+                      </PaginationLink>
+                    </PaginationItem>
```

Comment:
sorry just noticed, what is this button supposed to do? because currently it just jumps to the top of the page. 
I think it should either not be clickable, or a few other page numbers should be shown and clicking on them would turn to that page